addons/website_forum/models/forum_post.py

```diff
@@ -49,9 +49,10 @@ class Post(models.Model):
     create_uid = fields.Many2one('res.users', string='Created by', index=True, readonly=True)
     write_date = fields.Datetime('Updated on', index=True, readonly=True)
     bump_date = fields.Datetime('Bumped on', readonly=True,
-                                help="Technical field allowing to bump a question. Writing on this field will trigger "
-                                     "a write on write_date and therefore bump the post. Directly writing on write_date "
-                                     "is currently not supported and this field is a workaround.")
+                                help="Field to keep track of a post last activity."
+                                     "Updated whenever a reply, or a comment to the post or one of"
+                                     "its reply is added.",
+                                required=True, default= fields.Datetime.now())
```

Comment:
```suggestion
                                help="Field to keep track of a post's last activity. Updated whenever it is manually bumped, "
                                     "replied to, or when a comment is added on the post or one of its replies.",
                                required=True, default= fields.Datetime.now())
```

Please also see if we can't be more compact by going to the start of the line with the arguments instead of hanging here ^^

Reply:
up :)

Reply:
Oh BTW you forgot to update
```diff
-default= fields.Datetime.now()
+default=fields.Datetime.now
```
But I've got you covered  ;)